### PR TITLE
Opt-in for ExperimentalNativeApi to fix aggregate build

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

**Comment by qwwdfsad** [comment]
Please target `develop` when it's possible to reduce the number of commit in "aggregate only" branches one has to keep track of

**Comment by ilya-g** [comment]
@qwwdfsad we don't use develop branch here

**Comment by qurbonzoda** [comment]
All but the "Update K/N targets" commit will break the `master` branch. 
Also, we agreed on a slightly different workflow with the release team.

**Comment by qwwdfsad** [comment]
>@qwwdfsad we don't use develop branch here

Then it should be the master. The key is to reduce the number of commits that do not exist in our main branches -- the time has proven it's the source of regressions.


>All but the "Update K/N targets" commit will break the master branch.

In that scenario, it is recommended to `if` it in the build script using `1.9` LV or Kotlin version condition

**Comment by qurbonzoda** [comment]
I have scaled down the changes a bit and targeted the `master` branch. PTAL.

**Review by ilya-g — APPROVED** [review]

**Review by woainikk — APPROVED** [review]

File: integration/src/test/resources/templates/kotlin-multiplatform/build.gradle [hunk 1]
```diff
@@ -15,9 +15,6 @@ kotlin {
 
     sourceSets {
         commonMain {
-            dependencies {
-                implementation(benchmarkRuntimeMetadata)
```
[the remark this answers is not in the document]
Comment (qurbonzoda) [reply integration/src/test/resources/templates/kotlin-multiplatform/build.gradle]: I will address this concern in a separate PR

**Pull request merged**